src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -4,9 +4,9 @@
 import static seedu.address.commons.core.Messages.MESSAGE_INVALID_PROJECT_DISPLAYED_INDEX;
 import static seedu.address.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
 import static seedu.address.logic.commands.CommandTestUtil.ADDRESS_DESC_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.DEADLINE_DESC_A;
```

Comment:
I think we'd better keep the format aligned, say use `DEADLINE_DESC_AMY` as the name. And we can change `AMY` to project names together for future refactoring work.